hikari/impl/special_endpoints.py

```diff
@@ -1191,9 +1206,7 @@ async def create(
         *,
         guild: undefined.UndefinedOr[snowflakes.SnowflakeishOr[guilds.PartialGuild]] = undefined.UNDEFINED,
     ) -> commands.ContextMenuCommand:
-        return await rest.create_context_menu_command(
-            application, self._type, self._name, guild=guild, default_permission=self._default_permission
-        )
+        return await rest.create_context_menu_command(application, self._type, self._name, guild=guild)
```

Comment:
Don't you need to pass through `default_member_permissions` and `dm_enabled` here?

Reply:
Nice catch, I completely forgot to do so. Fixed now!